dora/src/semck/nameck.rs

```diff
@@ -359,7 +365,7 @@ mod tests {
         err(
             "fun Int() {}",
             pos(1, 1),
-            SemError::ShadowClassConstructor("Int".into()),
+            SemError::ShadowModule("Int".into()),
```

Comment:
Should we have a separate error for this: e.g. `ShadowClassAndModule`?

Reply:
I think it is ok in this case, as the problem is really the module, not the class.

Reply:
Even when the class has a constructor?